src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let basePrice = days * 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (SHORT_TERM <= days && days < LONG_TERM) {
+    return (basePrice -= SHORT_TERM_DISCOUNT);
```

Comment:
There is no need to use an assignment (`-=`) inside the return statement. You can simply return `basePrice - SHORT_TERM_DISCOUNT`.